websites/V/Vostfree/presence.ts

```diff
@@ -0,0 +1,102 @@
+const presence = new Presence({
+    clientId: "925799512205844490"
+  }),
+  path = document.location.pathname,
+  browsingStamp = Math.floor(Date.now() / 1000),
+  search = document.querySelector("#dle-content > h2"),
+  getGenreTitle = (genre: string) => {
+    [, , genre] = genre.split("/");
+    return genre;
+  };
+
+let video = {
+  duration: 0,
+  currentTime: 0,
+  paused: true
+};
+
+presence.on(
+  "iFrameData",
+  (data: { duration: number; currentTime: number; paused: boolean }) => {
+    if (data?.duration ? true : false) video = data;
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: browsingStamp
+    },
+    [privacy, button] = await Promise.all([
+      presence.getSetting<boolean>("privacy"),
+      presence.getSetting<boolean>("button")
+    ]),
+    title = document.querySelector(
+      "#dle-content > div.watch-top > div > div > div > div.slide-middle > h1"
+    );
+
+  if (!isNaN(video.duration) && title) {
+    const episode = document.querySelector(
+        "#player-tabs > div.tab-blocks > div:nth-child(1) > div > div.new_player_top > div.new_player_selector_box > div.jq-selectbox-wrapper > div > div.jq-selectbox__select > div.jq-selectbox__select-text"
+      ),
+      [startTimestamp, endTimestamp] = presence.getTimestamps(
```

Comment:
so you don't need L48-49
```suggestion
      [presenceData.startTimestamp, presenceData.endTimestamp] = presence.getTimestamps(
```

Reply:
I got the "Cannot redeclare block-scoped variable 'presenceData'." error

Reply:
Ooops, forgot aobut that. Replace the `,` on L41 and use a `;` instead